app/models/bobswatches_item.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
 
 class BobswatchesItem < Item
+  store_accessor :features, :box_and_papers, :year, :gender, :condition, :regular_price
```

Comment:
as I  can see `features` exists in all 3 shops, so we can use it as separate field

Reply:
also I don't see validations for items

Reply:
1. The store_accessor tells rails what pseudo columns we want to use via the :features hstore column.
2. I can't add any validations, because in all fields (except price) the data is represented differently everywhere, also can be nil.